packages/account/src/Sections/Security/ApiToken/api-token.jsx

```diff
@@ -51,8 +51,10 @@ class ApiToken extends React.Component {
 
         if (!token_name) {
             errors.token_name = localize('Please enter a token name.');
-        } else if (token_name.length < 2 || token_name.length > 32) {
+        } else if (token_name.length < 2) {
```

Comment:
constants?

```javascript
MIN_TOKEN = 2
MAX_TOKEN = 32
and replace the numbers above and below with these?
```

Reply:
@oskar-binary Thanks. I don't find any other place that uses these numbers, so I placed them in the file itself